compiler/pkg/parser/node_parser_test.go

```diff
@@ -64,18 +64,6 @@ var _ = Describe("Node parser tests", func() {
 		Expect(fail).To(BeTrue())
 	})
 
-	It("should fail when nexus child or link fields is an array.", func() {
```

Comment:
What caused the deletion of this test case?

Reply:
test case was already there 
https://github.com/vmware-tanzu/graph-framework-for-microservices/blob/795eaa238ec218a7a6a0b695f206a05c39dd1fac/compiler/pkg/parser/node_parser_test.go#L67

and I also I added one similar test case 

https://github.com/vmware-tanzu/graph-framework-for-microservices/blob/795eaa238ec218a7a6a0b695f206a05c39dd1fac/compiler/pkg/parser/node_parser_test.go#L91

Reply:
@PavitraS  PTAL